include/service/entities/template_service.js

```diff
@@ -110,12 +110,7 @@ module.exports = function(pb) {
          * @property reprocess
          * @type {Boolean}
          */
-        if(pb.config.TemplateService.useReprocess){
-            this.reprocess = true;
```

Comment:
... you deleted your own stuff. I dont think you meant to do that. may want to pull from hackathon.